packages/f-checkout/src/components/Checkout.vue

```diff
@@ -22,6 +22,14 @@
                 type="post"
                 :class="$style['c-checkout-form']"
                 @submit.prevent="onFormSubmit">
+                <p
+                    :class="[
+                        $style['c-checkout-switchUserLink']
+                    ]"
+                    data-test-id="create-account-login-link"
+                    @click="visitLoginPage">
+                    <a :href="loginUrl">Not {{ name }}? Click here.</a>
```

Comment:
@BanksySan, could you ensure this is coming from a translation file? So that we don't have to change it in the future.